mpris2controller

```diff
@@ -207,6 +208,26 @@ class Controller(dbus.service.Object):
         log.info('Method call for Previous.')
         self.call_on_one_playing('Previous')
 
+    @dbus.service.method(dbus_interface=MY_INTERFACE)
+    def Status(self):
+        log.info('Method call for Status.')
+        status = ''
+        if self.playing:
+            status = Player(self.bus, list(self.playing)[0]).get('PlaybackStatus')
+        elif self.not_playing:
+            status = Player(self.bus, self.not_playing[-1]).get('PlaybackStatus')
+        return status
+
+    @dbus.service.method(dbus_interface=MY_INTERFACE)
+    def Metadata(self):
+        log.info('Method call for Metadata.')
+        metadata = ''
```

Comment:
Would this return type be consistent with `.get('Metadata')`? I thought the latter was dict-like.

Reply:
No, `.get('Metadata')` is of type `dbus.Dictionary(..., signature=dbus.Signature('sv'), variant_level=1)`, while `metadata = ''` is of type `str`.

Now, as you pointed this out, I think it could be more clear to change it to `metadata = None` to represent situation where there is no(ne) metadata because of lack of launched media players.

Or it could be the empty `dbus.Dictionary`.

Reply:
I think `metadata = None` probably makes more sense here then

Reply:
I also changed it for the `Status`, and added comments for extra explanation.